infrastructure/validators/rewards-payout.md

```diff
@@ -0,0 +1,216 @@
+
+Note that while DOT is used as an example, this same formula would apply to KSM when running a
+validator on Kusama.
+
+```
+Validator Set Size (v): 4
+Validator 1 Stake (v1): 18 DOT <- Your validator
+Validator 2 Stake (v2):  9 DOT
+Validator 3 Stake (v3):  8 DOT
+Validator 4 Stake (v4):  7 DOT
+Payout (p): 8 DOT
+
+Your payout = (p / v) * 1 = (8 / 4) * 1 = 2
+```
+
+Running two validators, and splitting the stake equally, would result in the original validator `v4`
+to be kicked out of the validator set, as only the top `v` validators (as measured by stake) are
+selected to be in the validator set. More important, it would also double the reward that you get
+from each era.
+
+```
+Validator Set Size (v): 4
+Validator 1 Stake (v1): 9 DOT <- Your first validator
+Validator 2 Stake (v2): 9 DOT <- Your second validator
+Validator 3 Stake (v3): 9 DOT
+Validator 4 Stake (v4): 8 DOT
+Payout (p): 8 DOT
+
+Your payout = (p / v) * 2 = (8 / 4) * 2 = 4
+```
+
+With enough stake, you could run more than two validators. However, each validator must have enough
+stake behind it to be in the validator set.
+
+The incentives of the system favor equally-staked validators. This works out to be a dynamic, rather
+than static, equilibrium. Potential validators will run different numbers of validators and apply
+different amounts of stake to them as time goes on, and in response to the actions of other
+validators on the network.
+
+## Nominators and Validator Payments
+
+A nominated stake allows you to "vote" for validators and share the rewards (but also
+[slashing](../learn/learn-offenses.md)) without running a validator node yourself.
+
+Although staking rewards are based on the activities of the validator node during a specific era,
+the validator never has access to or ownership of staking rewards. In fact, `staking.payoutStakers`
+or `staking.payoutStakerByPage` calls are necessary to payout staking rewards, can be called by
+anyone, and the staking rewards are "generated" because of it and automatically sent to nominators
+(i.e., rewards are produced or minted and sent to nominators, not sent from validators to
+nominators).
+
+This includes the stake of the validator itself plus any stake bonded by nominators.
+
+!!!info
+    Validators set their preference as a percentage of the block reward, _not_ an absolute number of
+    DOT. Polkadot's block reward is [based on the _total_ amount at stake](../learn/learn-inflation.md#ideal-staking-rate). The
+    commission is set as the amount taken by the validator; that is, 0% commission means that the
+    validator does not receive any proportion of the rewards besides that owed to it from self-stake,
+    and 100% commission means that the validator operator gets all rewards and gives none to its
+    nominators.
+
+In the following examples, we can see the results of several different validator payment schemes and
```

Comment:
As you are using DOT in these examples, I'd add a small note or sentence that explains that these example are thought for polkadot relay chain that uses DOT as native token